README.md

```diff
@@ -243,6 +243,28 @@ add(3, 2)!
 
 By the way, to see DreamBerd in action, check out [this page](https://github.com/TodePond/DreamBerd/blob/main/LICENSE.md).
 
+## Loops
+
+Loops can be complicated. The `return` keyword simply returns you to the beginning.
+
+```js
+const var x = 1!
+function loop() => {
```

Comment:
I think `return!` should just go back one line, `return!!` two lines, `return!!!` three lines etc.

Reply:
That could open up a huge can of worms for ways to modify other keywords in the same way. Could be revolutionary.

But I also think there would still need to be a way to create a loop like I have here.